userScripts-mv3/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "manifest_version": 3,
+  "name": "User Scripts Manager extension",
+  "description": "Demonstrates the userScripts API and optional permission, in MV3.",
+  "version": "0.1",
+  "host_permissions": ["*://*/"],
+  "permissions": ["storage", "unlimitedStorage"],
+  "optional_permissions": ["userScripts"],
```

Comment:
When attempting to load this extension in Nightly I receive the following error:

> Uncaught (in promise) Error: Cannot request permission userScripts since it was not declared in optional_permissions

Channel: Nightly
Version: 135.0a1 (2024-12-30) (aarch64)
OS: macOS 15.1 (24B83)

Reply:
Oops, I missed [Rob's comment below](https://github.com/mdn/webextensions-examples/pull/576/files#r1890374839) indicating that you must currently enable `extensions.userScripts.mv3.enabled` in `about:config`.